modules/controller/rpc-controller.js

```diff
@@ -205,14 +220,50 @@ class RpcController {
 
                 for (let id of ids) {
                     let isAsset = false;
-                    let {assertionId} = await this.blockchainService.getAssetProofs(id);
+                    const { assertionId } = await this.blockchainService.getAssetProofs(id);
                     if (assertionId) {
                         isAsset = true;
                         id = assertionId;
                     }
+                    this.logger.emit({
+                        msg: id,
+                        Event_name: 'resolve_assertion_id',
+                        Operation_name: 'resolve_assertion_id',
+                        Id_operation: operationId,
+                    });
+                    this.logger.emit({
+                        msg: 'Started measuring execution of resolve local',
+                        Event_name: 'resolve_local_start',
+                        Operation_name: 'resolve_local',
+                        Id_operation: operationId,
+                    });
+
                     const nquads = await this.dataService.resolve(id, true);
+
+                    this.logger.emit({
+                        msg: 'Finished measuring execution of resolve local',
+                        Event_name: 'resolve_local_end',
+                        Operation_name: 'resolve_local',
+                        Id_operation: operationId,
+                    });
+
                     if (nquads) {
+                        this.logger.emit({
+                            msg: 'Started measuring execution of create assertion from nquads',
+                            Event_name: 'create_assertion_from_nquads_start',
```

Comment:
I think we should call this: resolve_create_assertion_start

Reply:
yes, good catch